packages/ETLoad.package/ETLObjectHeader.class/methodProperties.json

```diff
@@ -22,5 +22,4 @@
 		"objectHash" : "DS 5/26/2016 10:36",
 		"objectHash:" : "DS 5/26/2016 10:36",
 		"objectSizeInWords" : "DS 6/11/2016 16:13",
-		"objectSizeInWords:" : "DS 6/11/2016 16:13",
-		"printString" : "tcf 6/1/2016 14:52" } }
```

Comment:
Ah I see it was somehow added in a previous commit, no idea how, but it's gone again :).